src/components/Observations/details/index.tsx

```diff
@@ -1,21 +1,79 @@
+import { useMemo, useState } from 'react';
 import { useParams } from 'react-router-dom';
+import { Box } from '@mui/material';
+import { TableColumnType } from '@terraware/web-components';
 import { APP_PATHS } from 'src/constants';
 import strings from 'src/strings';
+import useDeviceInfo from 'src/utils/useDeviceInfo';
+import { useDefaultTimeZone } from 'src/utils/useTimeZoneUtils';
+import { getShortDate } from 'src/utils/dateFormatter';
+import { useLocalization } from 'src/providers';
+import { useAppSelector } from 'src/redux/store';
+import { searchObservationDetails } from 'src/redux/features/observations/observationDetailsSelectors';
+import Card from 'src/components/common/Card';
+import Table from 'src/components/common/table';
+import Search from 'src/components/Observations/search';
 import DetailsPage from 'src/components/Observations/common/DetailsPage';
+import ObservationDetailsRenderer from './ObservationDetailsRenderer';
+
+const columns = (): TableColumnType[] => [
+  { key: 'plantingZoneName', name: strings.ZONE, type: 'string' },
+  { key: 'completedTime', name: strings.DATE, type: 'string' },
+  { key: 'totalPlants', name: strings.PLANTS, type: 'number' },
+  { key: 'totalSpecies', name: strings.SPECIES, type: 'number' },
+  { key: 'plantingDensity', name: strings.PLANTING_DENSITY, type: 'number' },
+  { key: 'mortalityRate', name: strings.MORTALITY_RATE, type: 'number' },
+];
 
 export default function ObservationDetails(): JSX.Element {
   const { plantingSiteId, observationId } = useParams<{
     plantingSiteId: string;
     observationId: string;
   }>();
+  const { activeLocale } = useLocalization();
+  const { isMobile } = useDeviceInfo();
+  const defaultTimeZone = useDefaultTimeZone();
+  const [search, setSearch] = useState<string>('');
```

Comment:
Because it is local to this component, the search term will not persist to other pages. Not a problem I think. Making it common would involve lifting the state up to the `Observations/index.tsx` level

Reply:
Thanks for the call out. Drill down views search context is local, I believe. The monitoring plot view for instance will be searching monitoring plot names, not zone names. True that the top level and next level searches are zone names - not sure it is a requirement at this time. We can certainly lift it up for the common use-cases later if needed 👍

Reply:
I went ahead and lifted it up.. and passed it down to 1) root level data view and 2) one level down into the details drill-down. The search value is being preserved across these views (up and down). For next drill down I will avoid passing it down since we won't be searching zone names.
It just occurred to me, if user reloads the page, the search value is lost - user may expect it to be preserved if it was preserved across views. Wondering if we should capture search in the url params instead. Maybe something we should do across the app at some later time.